[Bug] 定时任务执行时，cc-connect崩溃了

### cc-connect Version

1.3.2

### Operating System

Windows (WSL)

### Agent Type

Claude Code

### Platform

N/A

### Installation Method

npm (npm install -g cc-connect)

### Bug Description

time=2026-04-23T17:57:00.000+08:00 level=INFO msg="cron: executing job" id=95a9dc61 project=my-vault1 prompt=查询今天的天气预报，并将结果通过飞书消息发送给我（Obsidian仓库：C:/Users/Administrator/D...
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x7ff70ceff04a]

goroutine 78 [running]:
github.com/chenhg5/cc-connect/core.(*workspacePool).GetOrCreate(0x0, {0xc0002a6360, 0x2f})
        /root/code/cc-connect/core/workspace_state.go:112 +0x4a
github.com/chenhg5/cc-connect/core.(*Engine).getOrCreateWorkspaceAgent(0xc0002bc008, {0xc0002a6360, 0x2f})
        /root/code/cc-connect/core/engine.go:2200 +0x6e
github.com/chenhg5/cc-connect/core.(*Engine).ExecuteCronJob(0xc0002bc008, 0xc0002cc000)
        /root/code/cc-connect/core/engine.go:1075 +0xd07
github.com/chenhg5/cc-connect/core.(*CronScheduler).executeJob.func1()
        /root/code/cc-connect/core/cron.go:648 +0x25
created by github.com/chenhg5/cc-connect/core.(*CronScheduler).executeJob in goroutine 87
        /root/code/cc-connect/core/cron.go:647 +0x331

### Steps to Reproduce

如上

### Expected Behavior

如上

### Actual Behavior

如上

### Configuration (config.toml)

```toml

```

### Logs / Error Output

```shell

```

### Additional Context

_No response_

--- a/core/engine.go
+++ b/core/engine.go
@@ -2197,6 +2197,9 @@ func (e *Engine) processInteractiveMessageWith(p Platform, msg *Message, session
 // getOrCreateWorkspaceAgent returns (or creates) a per-workspace agent and session manager.
 // workspace must be a normalized path (from resolveWorkspace or normalizeWorkspacePath).
 func (e *Engine) getOrCreateWorkspaceAgent(workspace string) (Agent, *SessionManager, error) {
+	if e.workspacePool == nil {
+		return nil, nil, fmt.Errorf("workspace pool not initialized (multi-workspace mode not enabled)")
+	}
 	ws := e.workspacePool.GetOrCreate(workspace)
 	ws.mu.Lock()
 	defer ws.mu.Unlock()

